tests/data/html/featureimage.html

```diff
@@ -1 +1 @@
-<figure class="featured-image"><picture><img src="https://placekitten.com/200/301" alt="Brown Kitten Image"><picture><figcaption> Cute Kitty </figcaption></figure>
\ No newline at end of file
+<figure class="featured-image"><picture><img src="https://placekitten.com/200/301" alt="Brown Kitten Image"></picture></figure>
```

Comment:
please confirm with Pushkraj once, if we are changing it here then its CSS name also need to change.

Reply:
@deepaksisodiya ok, though the output is already changed as suggested by @cryogenic-ric. Here is the PR #24